packages/playground/src/components/app-root/app-root.tsx

```diff
@@ -271,6 +271,26 @@ export class AppRoot {
     return loggedUser;
   }
 
+  async deleteAccount(): Promise<void> {
+    const token = window.localStorage.getItem(
+      "playground:user:token"
+    ) as string;
+
+    if (!token) {
+      return;
+    }
+
+    const user = await PlaygroundAPIClient.getUser(token);
+
+    try {
+      await PlaygroundAPIClient.deleteAccount(user);
+    } finally {
+      this.updateAccount({ hasCorruptStateChannelState: false });
```

Comment:
This should come immediately after the `deleteAccount` has successfully returned instead of being in the finally block.